### DISCUSSION TITLE
How to Blur the applications?

### DISCUSSION THREAD

**Discussion by MeditatedBison** [end-4/dots-hyprland#434] [Q&A] [answered]
I did `windowrulev2 = opacity 0.8 opacity override, class:^(APPNAME)`
but all it does it make the app transparent, Not the blur effect in it. 
i changed `special = true` in the Decoration > Blur section of general.conf. So now i can have blur in the application transparency, but only in special workspaces. Not in normal ones... How do i do that in normal ones too?

    blur {
        enabled = true
        xray = true
        special = true 
        new_optimizations = true
        size = 7
        passes = 4
        brightness = 1
        noise = 0.01
        contrast = 1
        popups = true
        popups_ignorealpha = 0.6
    }

![image](https://github.com/end-4/dots-hyprland/assets/95769851/ae62b539-a369-43f6-839e-80b0ca4b3093)

Foot terminal in special workspace:
![image](https://github.com/end-4/dots-hyprland/assets/95769851/59f84e31-8d00-4b21-86b7-cbbf76fa00b5)

**Comment by clsty** [comment, marked as answer]
https://end-4.github.io/dots-hyprland-wiki/en/i-i/02usage/#enabling-blur-for-apps

**Reply by nihalxkumar** [reply]
> [end-4.github.io/dots-hyprland-wiki/en/i-i/02usage#enabling-blur-for-apps](https://end-4.github.io/dots-hyprland-wiki/en/i-i/02usage/#enabling-blur-for-apps)

Updated link: https://end-4.github.io/dots-hyprland-wiki/en/i-i/03config/#enabling-blur-for-apps
